util/chunk/row.go

```diff
@@ -88,7 +89,8 @@ func (r Row) GetTime(colIdx int) types.Time {
 // GetDuration returns the Duration value with the colIdx.
 func (r Row) GetDuration(colIdx int) types.Duration {
```

Comment:
Does this change affect the benchmark of `GetDuration`?

Reply:
I will benchmark later, but it seems will allocate more a `types. Duration` than before, but this is just `GetTime` does...